proca/lib/proca_web/schema/eci_schema.ex

```diff
@@ -17,6 +17,7 @@ defmodule ProcaWeb.Schema.EciSchema do
   import_types(ProcaWeb.Schema.OrgTypes)
   import_types(ProcaWeb.Schema.UserTypes)
   import_types(ProcaWeb.Schema.SubscriptionTypes)
+  import_types(ProcaWeb.Schema.TargetTypes)
```

Comment:
eci shouldn't be changed, they will never have targets

Reply:
The types wouldn't load without this. Marcin said it was required to import it here.

Reply:
Yes eci does not use this type in schema but absinthe will report missing type